### ISSUE TITLE
PF4 Brand alt text

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#908]
The alt text in the brand component should be updated to match the example we have in core.

Currently, the react component uses the following alt text:
`alt="Brand Image" `
Whereas the example in core uses:
`alt="Patternfly Logo"`

Whenever text is included in an image, it should be repeated in the `alt` attribute value (this is how screen readers know what the text is). Additionally, alt text can describe the image (e.g. that it's a "logo").  

The Additional Notes section at the end of [this article](https://www.phase2technology.com/blog/no-more-excuses-definitive-guide-alt-text-field) provides a good checklist that's relevant for logos.
